src/network/graphQuery.js

```diff
@@ -186,14 +186,20 @@ export function queryAllVotes(filters, orderBy) {
 * @param filters {Array} Array of objects for filtering. ie. [{ status: 'WAITRESULT' }, { status: 'OPENRESULTSET' }]
 * @param orderBy {Object} Object with order by fields. ie. { field: 'blockNum', direction: 'DESC' }
 */
-export function queryAllTransactions(filters, orderBy) {
+export function queryAllTransactions(filters, orderBy, limit, skip) {
   const request = new GraphQuery('allTransactions', TYPE.transaction);
   if (!_.isEmpty(filters)) {
     request.setFilters(filters);
   }
   if (!_.isEmpty(orderBy)) {
     request.setOrderBy(orderBy);
   }
+  if (_.isFinite(limit) && limit > 0) {
```

Comment:
why is finite not !isEmpty?

Reply:
`isFinite` is checking if it is a finite number (non-theoretical) and also for undefined too. as most lodash checks do.

`limit` is a number so should be an undefined or > check.